src/draw/stm32_dma2d/lv_gpu_stm32_dma2d.c

```diff
@@ -11,100 +11,75 @@
 
 #if LV_USE_GPU_STM32_DMA2D
 
+#include "stdio.h"
+#include "main.h"
```

Comment:
It should be removed.

Reply:
That's right both those includes (stdio.h and main.h) can be removed, but `LV_GPU_DMA2D_CMSIS_INCLUDE` needs to be set to (e.g.) "stm32f7xx_hal.h", rather than "stm32f769xx.h". This way constants like `DMA2D_INPUT_ARGB8888` can be included from HAL, rather than redefined in LVGL. Consequently, it would probably make sense to rename `LV_GPU_DMA2D_CMSIS_INCLUDE` to `LV_STM32_HAL_INCLUDE` or similar. What do you think?

Reply:
It worked thanks.